functions/Backup-DbaDatabase.ps1

```diff
@@ -609,10 +621,24 @@ function Backup-DbaDatabase {
             }
 
             if ($True -eq $ReplaceInName) {
+                # The following is needed as a piped in database will provide servername wrapped in []'s
```

Comment:
No. `$server` is an object of type Server. Only it's .ToString() surrounds it by []. If you need just the Name, that is in `.Name` (but if it is a DAC, it would include the "ADMIN" - so better not use this). You can use `.DomainInstanceName`to get ComputerName and optionally InstanceName - like we do in the output property SqlInstance.
![image](https://user-images.githubusercontent.com/66946165/152746605-f136e215-6bba-446e-944d-41307afba52a.png)

Reply:
![image](https://user-images.githubusercontent.com/66946165/152747554-790e414e-152a-4c47-aec1-77cdca693aa2.png)